src/k8s/components/indexed-search/configuration/toInternal.dhall

```diff
@@ -64,7 +71,11 @@ let Container/Zoekt-Indexserver/toInternal
 
         let envVars = environment/toList opts.environment
 
-        let envVars = Some (envVars # opts.additionalEnvVars)
+        let envVars =
```

Comment:
Ran `envVars` through utility functions to have it be `None` if both lists are empty

Reply:
makes sense

Reply:
good idea, nice